internal/handlers/initrd.go

```diff
@@ -67,6 +68,21 @@ func (h *initrdHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 	defer initrdReader.Close()
 
+	ramdisk, statusCode, err := h.client.ramdiskContent(r, imageID)
```

Comment:
Sorry, I might be missing something, but won't this get all of the minimal initrd as opposed to just the static network config files?

Reply:
This could all use better names at this point.

This will get the initrd from assisted which contains the static network config files. The same one that we embed in the minimal iso.

You can see how it's created [here](https://github.com/openshift/assisted-service/blob/e69c126e90b94eff81a193a2b8fd132451971bd2/internal/bminventory/inventory.go#L3381-L3415). It's pretty much just the static network config.

Reply:
Gotcha, thanks for the clarification!